GitHub-PR-Tips.md

```diff
@@ -0,0 +1,197 @@
+# GitHub & Pull Request (PR) Tips
+
+As of May 2024, the TianoCore project transitioned from a mailing list based contribution process to a pull request
+process. Since many community developers may not be familiar with essential GitHub and pull request features, this guide
+is meant to show various tips to improve your workflow making it easier to contribute and review changes in edk2.
+
+1. [GitHub UI key features](#github-ui-key-features)
+   - [Forking the edk2 repository](#forking-the-edk2-repository)
+   - [Setting up notifications for the edk2 repository](#setting-up-notifications-for-the-edk2-repository)
+   - [Creating a Pull Request](#creating-a-pull-request)
+   - [Viewing Changes in a Pull Request](#viewing-changes-in-a-pull-request)
+   - [Leaving Feedback in a Pull Request](#leaving-feedback-in-a-pull-request)
+   - [Viewing Open & Closed Pull Requests](#viewing-open--closed-pull-requests)
+   - [Finding a commit](#finding-a-commit)
+   - [Finding a Pull Request for a commit](#finding-a-pull-request-for-a-commit)
+   - [Contributing entirely in your Web browser](#contributing-entirely-in-your-web-browser)
+
+2. [VS Code GitHub PR plugin](#2-vs-code-github-pr-plugin)
+
+3. [Official GitHub CLI tool](#3-official-github-cli-tool)
+
+4. [Official GitHub Desktop application](#4-official-github-desktop-application)
+
+## GitHub UI Key Features
+
+### Forking the edk2 Repository
+

```

Comment:
Copying an earlier comment made by @mdkinney [here](https://github.com/makubacki/tianocore.github.io/commit/2c1a3f29c9da368a9f33fa6cfae07f8f6120f17e#r142054262):

> Lets add some information that the TianoCore contribution process requires the use of a fork. No developers have direct write access to the TianoCore repos, so branches with proposed changes must be created in a fork.

Reply:
I will add that change.

Reply:
Done